static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +53,41 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
+
+If `less` is not runnable (usually Microsoft Windows), the ouput is simply
+printed out.
+
+> Note Microsoft Windows users may also want to read
+> [this how-to](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+### Providing a custom pager
+
+You can also override the default pager via `DVC_PAGER` environemnt variable.
+
+For example:
+
+Following command will replace the default pager with
+[`more`](<https://en.wikipedia.org/wiki/More_(command)>) for a single call
```

Comment:
It's possible to override the default pager via the `DVC_PAGER` environment variable. For example, the following command will replace the default pager with [`more`](<https://en.wikipedia.org/wiki/More_(command)>), for a single run: